library/CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ list(APPEND src_crypto ${thirdparty_src})
 set(src_x509
     certs.c
     pkcs11.c
+    pkcs7.c
```

Comment:
Since there's a new source file, it needs to be listed in `visualc/VS2010/mbedTLS.vcxproj`. Please run `tests/scripts/check-generated-files.sh -u` and commit the result, or run the update scripts individually and make sure that `tests/scripts/check-generated-files.sh` is happy.

This is why some Windows jobs are failing on CI. When the library is fully built, this may reveal further errors (Visual Studio often has warnings in places where GCC doesn't).

Reply:
This is no longer relevant: the generated files aren't committed any more since 3.0.